src/components/Person/PersonLink.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const { slugParam } = useParams();
+
+  const { sex, born, died, fatherName, motherName, name, slug } = person;
+  const hasMother = people.find(p => p.name === person.motherName);
+  const hasFather = people.find(p => p.name === person.fatherName);
```

Comment:
I think we need to find a better name for these variables hasMother is more suitable for a function than for a constant variable